jpa/eclipselink/pom.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+	<parent>
+		<groupId>org.springframework.data.examples</groupId>
+		<artifactId>spring-data-jpa-examples</artifactId>
+		<version>1.0.0.BUILD-SNAPSHOT</version>
+	</parent>
+
+	<repositories>
```

Comment:
Okay, let's stick to plugin then. Is it possible to at least use a non-snapshot version of it or is the classpath scanning only available in 1.1? If in doubt, keep Boot customizations and Spring setup to a minimum and move custom setup into the build, so that we expose the infrastructure setup cost that way.

Does that make sense and provide enough guideline?